### ISSUE TITLE
Add support for leading spaces in search pattern

### ISSUE THREAD

**Issue by sanderr** [c4urself/bump2version#199]
Leading spaces are currently trimmed from the search pattern. I believe this is inherent in the library used to parse the config file. Since this effectively makes bumpversion unusable for any projects that need a search pattern with leading spaces, I think it would be valuable to add space (and potentially tab) to the special character context so it can be used similarly to `{#}`.

**Comment by florisla** [comment]
I think this makes sense.  Where did you find out about `{#}`? I was not familiar with that.

**Comment by sanderr** [comment]
I found out about it by digging into the code when I was trying to find out the strange behavior that resulted in #198. See [here](https://github.com/c4urself/bump2version/blob/190a9940a6190401a4e528d832d9a0e82e0a49c2/bumpversion/cli.py#L60).

**Comment by florisla** [comment]
We should document this in the README too.

What keywords do we use for space and tab? Simply `{space}` and `{tab}`?

**Comment by sanderr** [comment]
I agree documenting this would be a good idea. I believe there were some other things I didn't find in the README but I didn't think to write them down. I'll create another ticket if it comes to mind.

I like `{space}` and `{tab}`, I think they clearly convey the intention.

**Comment by kdeldycke** [comment]
This issue has been solved by [`bump-my-version`](https://github.com/callowayproject/bump-my-version) thanks to its transition to TOML-based configuration files.
